src/applications/personalization/profile360/components/DowntimeBanner.jsx

```diff
@@ -1,22 +1,19 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
-import { externalServiceStatus } from '../../../../platform/monitoring/DowntimeNotification';
+import { externalServiceStatus } from 'platform/monitoring/DowntimeNotification';
 
 function DowntimeBanner({ downtime, section }) {
   return (
     <AlertBox
       status="warning"
       isVisible
+      headline={`We can’t show your ${section} information right now.`}
```

Comment:
Moving the H3 content to here fixes an issue with the headline rendering too low in the AlertBox